extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -519,7 +519,7 @@ function arrayslice( &$parser, $arrayid_new , $arrayid , $offset, $length='') {
 *    {{#arrayintersect:arrayid_new|arrayid1|arrayid2}}
 *   See: http://www.php.net/manual/en/function.array-intersect.php
 */
-    function arrayintersect( &$parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
+    function arrayintersect( $parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
         if (!isset($arrayid_new) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)